src/templates/rest-api/src/app.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022, Ansys Inc. Unauthorised use, distribution or duplication is prohibited
+
+from flask import Flask, jsonify, request
+from flask_cors import CORS, cross_origin
+from flask_swagger_ui import get_swaggerui_blueprint
+import argparse
+import logging
+from src.blueprints.version import blueprint as version_endpoint
+from src.blueprints.health import blueprint as health_endpoint
+
+
+SWAGGER_URL = '/api/docs'
+API_URL = '/static/swagger.json'
+
+
+SWAGGER_UI_BLUEPRINT = get_swaggerui_blueprint(
+    SWAGGER_URL,
+    API_URL,
+    config={
+        'app_name': "API Name"
+    }
+)
+
+
+def create_app():
+    """Initialize the core application"""
+    app = Flask(__name__)
+    cors = CORS(app)
+    app.config['CORS_HEADERS'] = 'Content-Type'
+    app.register_blueprint(version_endpoint)
+    app.register_blueprint(health_endpoint)
+    app.register_blueprint(SWAGGER_UI_BLUEPRINT, url_prefix=SWAGGER_URL)
+
+    return app
+
+
+def serve(app, address, port, middleware=None):
+
+    if middleware is not None:
+        middleware(app)
+
+    logging.info('Server flask starting')
+    app.run(host=address, port=port)
+    i=1
```

Comment:
What is the `i = 1` for? it doesn't do anything

Reply:
typo

Reply:
Hey @asbfall you never fixed this before merging it in!